Example/Sources/ViewControllers/UserViewController.swift

```diff
@@ -101,32 +101,35 @@ extension UserViewController: UITableViewDataSource {
     cell.accessoryType = .disclosureIndicator
     return cell
   }
-
+  
 }
 
 
 // MARK: - UITableViewDelegate
 
 extension UserViewController: UITableViewDelegate {
-
+  
   func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
     tableView.deselectRow(at: indexPath, animated: false)
     let repo = self.repos[indexPath.row]
     let webViewController = Navigator.present(repo.URLString, wrap: true)
     webViewController?.title = "\(self.username)/\(repo.name)"
     NSLog("Navigator: Present \(repo.URLString)")
   }
-
+  
 }
 
 
 // MARK: - URLNavigable
 
-extension UserViewController: URLNavigable {
+typealias NavigableFunction = (Any) ->Void
```

Comment:
Want to use the newer initialize method in demo. so pass `NSObject()` with it. Remove the useless typealias